README.md

```diff
@@ -1,15 +1,33 @@
-# CNN Annotator
+# Convolutional neural networks 🖥️ for classifying chromatin morphology 🧬 in live cell imaging 🔬
+<br/>
 
-### Convolutional neural networks for classifying chromatin morphology in live cell imaging
+<img width="50%" align="right" alt="Protocol" src="assets/napari_annotator.png" />
 
+Welcome to our [CNN Annotator repository](https://github.com/lowe-lab-ucl/cnn-annotator "CNN Annotator repository || Lowe Lab UCL") where you can find our recent *Methods to Molecular Biology (MiMB) protocol* on how to use a convolutional neural network (CNN) to classify time-lapse microscopy single-cell images patches according to live-cell chromatin morphology.
+
+**Wondering what such pipeline could be used for?** Check out our recent publications where we made use of CNNs for classifying cells in live-cell imaging:
+- [Ulicna *et al.*, bioRxiv, 2020](https://www.biorxiv.org/content/10.1101/2020.09.10.276980v1.full "Automated deep lineage tree analysis using a Bayesian single cell tracking approach") - Automated deep lineage tree analysis using a Bayesian single cell tracking approach
+- [Bove *et al.*, MBoC, 2017](https://www.molbiolcell.org/doi/10.1091/mbc.E17-06-0368?url_ver=Z39.88-2003&rfr_id=ori:rid:crossref.org&rfr_dat=cr_pub%20%200pubmed "Local cellular neighbourhood controls proliferation in cell competition") - Local cellular neighbourhood controls proliferation in cell competition
+
+
+### How to navigate this repository
+
+We provide a detailed walk-through for annotating live-cell microscopy images and training CNN model to infer classification labels on previously unseen images. Here is an overview of the entire process:
+
+![Protocol Pipeline](/assets/protocol_pipeline.png)
+
+For more detailed instructions on how to annotate your microscopy data, train the CNN classifier and infer labels on previously unseen images, please refer to [this step-wise manual](/notebooks/README.md "Protocol Methods").
+
+Please use these links to proceed with the training and inference of your CNN models in the Google Colab environment:
```

Comment:
This is great!

Reply:
Minor thing: @KristinaUlicna if you have time, can you add a white rectangle background to the 'protocol_pipeline.png'? When github is shown in dark mode, one can't see the black arrows. I figure some users will experience that.